### ISSUE TITLE
Does not work on Centos 7

### ISSUE THREAD

**Issue by khanc** [jlebon/textern#29]
Nothing happens on Centos 7. python36 is available in the system. Installed native app with USER=1, installed addon, trying ctrl-shift-E. Absolutely nothing happens.

**Comment by jlebon** [comment]
Can you check if there are any errors in the browser console? (Ctrl+Shift+J)

**Comment by khanc** [comment]
Sorry for the delay. Good news is I was able to go a bit further. The textern.py looks for python3 which doesnt exist in Centos 7, so I put a symbolic link to /bin/python3.6. So far so good.

However there is still trouble. The only editor I was able to successfully use was gedit. I want to use vim (not gvim). The konsole suggestion (https://github.com/jlebon/textern) does not work, it is opening a file in a directory which does not exist, which seems to be the same problem as using gnome-terminal --command. Then I tried to use https://raw.githubusercontent.com/jlebon/files/master/bin/gnome-terminal-wrapper, which also does not work (no error message, nothing on browser console). When I tried gnome-terminal-wrapper vim filename, I get the error:  
usage: new-session [-AdDP] [-F format] [-n window-name] [-s session-name] [-t target-session] [-x width] [-y height] [command]
which seems to be coming from tmux. I am at a loss how to fix this. Thanks for the help.
